### PR TITLE
SOX-265 test version identifiers after merge

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: SOX-265
KBC:  https://github.com/keboola/connection/pull/4530 

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -7630,7 +7630,8 @@ Gets configuration rows for [Development Branch](#reference/development-branches
                     "id": 27978,
                     "description": "name@example.com"
                 },
-                "changeDescription": "Row 236781529 added"
+                "changeDescription": "Row 236781529 added",
```
Comment (jirkasemmler) [inline apiary.apib:7633]: jsem zjistil, ze to neni v reponse examplech pro rows

File: tests/Backend/SOX/MergeRequestsTest.php [hunk 2]
```diff
@@ -797,6 +797,139 @@ public function testConfigurationUpdatedInBranch(): void
         $this->assertBranchIsDeleted($newBranch['id']);
     }
 
+    public function testVersionIdentifiersWhenCreatedConfigurationInBranch(): void
```
Comment (jirkasemmler) [inline tests/Backend/SOX/MergeRequestsTest.php:800]: dalo by se to spojit do jednoho testu, nebo k necemu prilepit ale preferuju spis, aby to bylo explicitne videt, oba usecasy. Jinak to zvysuje riziko, ze mozek samovolne opusti prostor lebky, familierne nazyvane jako "bouchne mi z toho hlava"

Comment (jirkasemmler) [reply tests/Backend/SOX/MergeRequestsTest.php:800]: hmm, tak na to koukam a vlastne jsem uplne zrcadlil testy testCreateConfigurationInBranch a testConfigurationUpdatedInBranch . ASi by bylo lepsi jim pridat check na changeLog a realizovat ten test tam, misto pridavani techto dovu

Comment (romanbracinik) [reply tests/Backend/SOX/MergeRequestsTest.php:800]: No jak chces uz by som to mozno nechal ked to mas napisane

**Review by romanbracinik — CHANGES_REQUESTED** [review]
doplnil by som ten $this->fail(), ostatne su len doporucenia :)

**Comment by jirkasemmler** [comment]
@romanbracinik tak jsem to nakonec pridal do tech existujicich testu - pridal jsem tam ten changelog a ono to ve vysledku testuje to same.

**Review by romanbracinik — APPROVED** [review]
🙆‍♂️

**Pull request merged**